### ISSUE TITLE
Duplicate logs are created when logproxy runs into an error

### ISSUE THREAD

**Issue by RuudMeeuwsen** [philips-software/logproxy#279]
In our Healthdot Kibana logging, we see certain logs being duplicated and others missing.

E.g. The following snippet of logging:
![image](https://user-images.githubusercontent.com/92581255/199724352-6644a198-c460-4e71-b8d3-e15dfc6c77c1.png)

Resulted in the following duplication in Kibana:
![image](https://user-images.githubusercontent.com/92581255/199724698-7051f50c-6605-4dcf-a36d-429c31271062.png)

It seems that when the duplication occurs, logproxy itself logs a sequence of messages such as:
![image](https://user-images.githubusercontent.com/92581255/199724012-ef255d85-0d70-4def-b55f-3de64be42ba2.png)

The error that occurred doesn't seem to be logged, so it is currently unclear what the root cause is.

**Comment by RuudMeeuwsen** [comment]
After the latest update, we no longer see the duplicate logs. However, we're still missing log entries.
Thanks to the additional logging that logproxy now provides, I was able to capture a few log entries that result in an error, and an example of a log entry that is being forwarded.

I've captured three POSTs that resulted in an error:
![logproxy_error1](https://user-images.githubusercontent.com/92581255/203264555-391a3b6e-89e1-4ea1-a735-8d77040043fc.png)
![logproxy_error2](https://user-images.githubusercontent.com/92581255/203264569-68844bce-6481-498f-bad4-d18bfc7d7459.png)
![logproxy_error3](https://user-images.githubusercontent.com/92581255/203264582-1e8315c6-1e3a-44f5-add2-fbb7ffd1e8d7.png)

While the response from HSP complains about the transactionId, that seems to be incorrect, as the transactionId is in the request and contains a UUIDv4 (which should be accepted by HSP Logging).

I've captured this example of a POST that was accepted:
![logproxy_good](https://user-images.githubusercontent.com/92581255/203264766-6b1e45fc-ab02-448a-8f76-a559dfff4f76.png)
Interestingly, this also contains a very similar transactionId, suggesting that that is indeed not the issue.
However, it also shows that in this example, the message is base64 encoded (which it was not in the previous examples).

Could it be that were running into two issues here?
- Logproxy not base64 encoding the message in some cases
- HSP Logging returning an incorrect error message in this scenario
